--- SConscript.orig	2017-08-31 13:47:25.206364234 +0000
+++ SConscript	2017-08-31 13:49:17.528260598 +0000
@@ -60,13 +60,11 @@
         'zookeeper_mt',
         'ssl',
         'crypto',
+        'dl',
+        'sasl2',
         'boost_filesystem',
         'boost_program_options'])
 
-(PLATFORM, VERSION, EXTRA) = platform.linux_distribution()
-if PLATFORM.lower() != 'ubuntu':
-    AnalyticsEnv.Prepend(LIBS=['sasl2'])
-
 database_libs = ['cassandra_cql', 'cassandra']
 
 database_libs.append('gendb')
